wdio.conf.android.js

```diff
@@ -1,3 +1,10 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
```

Comment:
please update the year to 2022

Reply:
still using 2021 in wdio.conf.js, that's I put 2021 here as well. It should be 2022 since adding it in 2022?